x/concentrated-liquidity/incentives.go

```diff
@@ -387,31 +372,18 @@ func (k Keeper) initOrUpdatePositionUptime(ctx sdk.Context, poolId uint64, posit
 			if err != nil {
 				return err
 			}
+			if recordExists {
```

Comment:
Above this line it says

```
// If a record does not exist for this uptime accumulator, create a new position.
// Otherwise, add to existing record.
```

This is no longer the case